ldelf/ta_elf_rel.c

```diff
@@ -81,7 +83,9 @@ static TEE_Result resolve_sym_helper(uint32_t hash, const char *name,
 		Elf64_Sym *sym = elf->dynsymtab;
 
 		for (n = bucket[hash % nbuckets]; n; n = chain[n]) {
-			assert(n < nchains);
+			if (n >= nchains)
+				err(TEE_ERROR_BAD_FORMAT,
+				    "Index out of range");
```

Comment:
minor: fits in a 80 char line.

Reply:
Yes, but only just it doesn't fit a 79 char line. I'd prefer to keep this unchanged.

Reply:
ok, yet [80 char is fine](https://www.kernel.org/doc/html/v5.4/process/coding-style.html#breaking-long-lines-and-strings).